components/admin/questDetails.tsx

```diff
@@ -24,9 +24,12 @@ import { useNotification } from "@context/NotificationProvider";
 import Button from "@components/UI/button";
 import { useRouter } from "next/navigation";
 import DownloadQuestParticipantsButton from './DownloadQuestParticipantsButton';
+import DownloadBoostWinnersButton from "./DownloadBoostWinnersButton";
+
 
 type QuestDetailsProps = {
   quest: QuestDocument;
+  boost: ClaimableQuestDocument;
```

Comment:
Don't add this new param. The boostId is quest.boosts[0].id. Hide the button if quest.boosts.length == 0

Reply:
In fact I think the best would be to map quest.boosts. 
Something like that:
```tsx
quest.boosts.map((boost) => 
<ButtonComponent>Download Boost {(boost.id)} Winners

Reply:
Hello Sir @Marchand-Nicolas, sorry for the misunderstood. In this part 
![image](https://github.com/user-attachments/assets/f08218c5-96d7-4bca-af62-0161202ce9d9)
that you told me the boostId is quest.boosts[0].id but the "quest: QuestDocument;" of "QuestDetailsProps" doesn't have that param, Do I have to add the param or I'm doing something wrong? I'm very sorry for the question.

Reply:
Ah you can test, I think if you try console.log(quest), you should see a boosts param in it. If so you can update the QuestDocument type

Reply:
I'm not sure if the new param of "boosts" in the QuestDocument type has to be something like this 
![image](https://github.com/user-attachments/assets/9e690a88-908d-4ea3-91c1-65070c2c70f7)
or like this 
![image](https://github.com/user-attachments/assets/34c4bb4c-bf91-440c-9670-2631b3171785)

Reply:
I think the first is better, if it contains an id field